conf/nginx-primaryonly-no-nextcloud.conf

```diff
@@ -0,0 +1,17 @@
+	# Control Panel
```

Comment:
Can a similar thing be done here with the nginx configuration files like I suggested with the fail2ban?  That is make a "base" config which the Nextcloud configuration is added to when necessary?